packages/react/src/contents/hooks/useContentType.ts

```diff
@@ -1,86 +1,82 @@
 import {
```

Comment:
The comments regarding `data` calculation, `refetch` action and the useEffect condition for the auto fetch that were raised for the `useCommercePages` hook also apply for this hook as well. To avoid repetition, can you please take a look at those points and change here as well?